### PR TITLE
281.ssl verify

### PR DESCRIPTION
**Pull request by d-w-moore**

### CODE REVIEW

File: irods/test/login_auth_test.py [hunk 1]
```diff
@@ -326,6 +344,41 @@ def test_8(self):
         self.tst0 ( ssl_opt = False, auth_opt = 'pam'    , env_opt = True  )
 
 
+class TestWithSSL(unittest.TestCase):
+    '''
+    The tests within this class should be run by other than the service account.
```
Comment (trel) [inline irods/test/login_auth_test.py:349]: "by other" -> "by an account other"

**Review by trel — COMMENTED** [review]
seems pretty good, just some comment tweaks

happy to just have things #'d as is or to squash a little bit - i'll leave it up to you.

File: irods/test/login_auth_test.py [hunk 2]
```diff
@@ -202,6 +221,8 @@ def tearDownClass(cls):
     def setUp(self):
         if not getattr(self, 'envdirs', []):
             self.skipTest('The test_rods_user "{}" does not exist'.format(self.test_rods_user))
+        if os.environ['HOME'] != '/var/lib/irods':
```
Comment (trel) [inline irods/test/login_auth_test.py:224]: does this consider non-package-install?    is that important in this scenario?    i think it's probably fine...

File: irods/test/login_auth_test.py [hunk 3]
```diff
@@ -326,6 +348,41 @@ def test_8(self):
...
 
 
+class TestWithSSL(unittest.TestCase):
+    '''
+    The tests within this class should be run by an account other than the
+    service account.  Otherwise there is risk of corrupting the server setup.
+    '''
+
+    def setUp(self):
+        if os.path.expanduser('~') == '/var/lib/irods':
```
Comment (trel) [inline irods/test/login_auth_test.py:358]: same question.

File: irods/test/login_auth_test.py (hunk 2 above)
Comment (d-w-moore) [reply irods/test/login_auth_test.py:224]: Ah, need to check if there's a way to get that.  iRODS API?   System package config won't work obvs....

File: irods/test/login_auth_test.py (hunk 3 above)
Comment (d-w-moore) [reply irods/test/login_auth_test.py:358]: Acknowledged, difficult to say.  the `login_auth_test.py` script needs to be run in two phases, once by the service account and once by a normal user.  Currently it only makes sense only  on a single node iRODS with a package install  and a service account.

Comment (trel) [reply irods/test/login_auth_test.py:358]: okay, that's fine for now then - let's just leave it as is - this is just testing code.

File: irods/test/login_auth_test.py (hunk 2 above)
Comment (trel) [reply irods/test/login_auth_test.py:224]: i don't think there is - the home directory / what user is running - is dictated by the OS...  let's leave it for now.

**Comment by trel** [comment]
i think this can now be #'d and we'll get it merged.

**Comment by d-w-moore** [comment]
Done !

**Pull request merged**